src/Controller/UserController.php

```diff
@@ -610,4 +611,164 @@ public function removeAvatar(
         }
         return new JsonResponse(['message' => 'Avatar successfully deleted!'], Response::HTTP_OK);
     }
+
+    /**
+     * Set Project Manager role
+     * @Rest\Post("/{id}/set_pm", requirements={"id"="\d+"})
```

Comment:
Role management should be general. Only admins can assign roles. An admin can promote anyone to PM or to admin and also demote to PM, user.